mewarpx/docs/changelog.rst

```diff
@@ -0,0 +1,12 @@
+=========
+CHANGELOG
+=========
+.. csv-table::
+   :widths: 5 5 85
+   :header-rows: 1
+   :file: ../changelog.csv
+
+.. csv-table::
+   :widths: 5 5 5 85
+   :header-rows: 1
+   :file: changelog_thru_v4.csv
```

Comment:
You can remove this. This comes from when we moved the `metools` repository into our `ME_Warp` repository.